### PR TITLE
Property test for TokenNetwork.

### LINKED ISSUE #153: Resolve property tests from raiden/[...]/property
Go over old [raiden/[...]/test_nettingchannel](https://github.com/raiden-network/raiden/blob/991275c9e8e9e4a4ccfdccb3a66a9e20950dd9b5/raiden/tests/property/smart_contracts/test_nettingchannel.py) and rebuild the tests that that are missing.  sub #92 

- [ ] assess if tests have to be rebuild
- [ ] if yes, rebuild them

### PR DESCRIPTION
**Pull request by rakanalh**
### What's been done:
1. Added the property test from an old revision of the Raiden repo.
2. Updated the setup to adapt to the new contracts structure.
3. Updated the calls to new contracts.

Fixes #153

### CODE REVIEW

File: raiden_contracts/tests/utils/__init__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# flake8: noqa
+
+from .address import *
+from .contracts import *
+from .utils import *
```
[the remark this answers is not in the document]
Comment (rakanalh) [reply raiden_contracts/tests/utils/__init__.py]: It's already done in fixtures:
https://github.com/raiden-network/raiden-contracts/blob/master/raiden_contracts/tests/fixtures/__init__.py

**Comment by rakanalh** [comment]
The test was disabled until this is implemented: https://github.com/raiden-network/raiden-contracts/issues/108.

This is due to the fact that we need a state machine test implementation which will trigger calls to the contracts with sampled data that can visit all execution paths of the contract.

**Comment by loredanacirstea** [comment]
I have not yet reviewed this PR. However, from some discussions with @rakanalh , I understand this test is not that useful.
Therefore: 
- why merge the PR if it is not that useful?
- why merge the PR and skip the test? What does this PR achieve then?
- why close the issue if it has not been solved? The test is skipped.

**Comment by LefterisJP** [comment]
> why merge the PR if it is not that useful?

Because this is just a skeleton for property tests. That's what I understood from the issue descriptions and discussion with Tobi and Rakan.

More works needs to be put to property tests to be useful and that should be reflected in another issue. After a quick look I can't find another issue @rakanalh. Did you make one?

**Comment by loredanacirstea** [comment]
I am ok with this if an issue is created with the contents from https://github.com/raiden-network/raiden-contracts/pull/181#issuecomment-405257389 . And referenced here. Otherwise, we might miss this in the future.

**Comment by err508** [comment]
The PR was linked to #108, which describes missing property-tests

**Comment by rakanalh** [comment]
I moved the comment Tobi and I made above to #108 to make sure the details of next steps aren't missed.

**Pull request merged**